App.js

```diff
@@ -26,7 +26,7 @@ import SettingsEmailScreen from './src/components/auth/SettingsEmailScreen'
 import SettingsPasswordScreen from './src/components/auth/SettingsPasswordScreen'
 
 // MainStack
-import HomeScreen from './src/components/main/HomeScreen'
+import HomeScreen from './src/components/main/HomeScreen_alt'
```

Comment:
If you're already replacing it, rename your file to HomeScreen.js and the old one to HomeScreen_Dep.js